### PR TITLE
Renovate - consume PE global configuration

### PR DESCRIPTION
**Pull request by lmilbaum**
### What does this PR do?:
Renovate to consume platform engineering global configuration

### Which issue(s) this PR fixes:
_Link to github issue(s)_

### PR acceptance criteria:

- [ ] Contributing guide
_Have you read the [devfile registry contributing guide](https://github.com/devfile/registry/blob/main/CONTRIBUTING.md) and followed its instructions?_
- [ ] Test automation
_Does this repository's tests pass with your changes?_
- [ ] Documentation
_Does any documentation need to be updated with your changes?_
- [ ] Check Tools Provider
_Have you tested the changes with existing tools, i.e. Odo, Che, Console? (See [devfile registry contributing guide](https://github.com/devfile/registry/blob/main/CONTRIBUTING.md) on how to test changes)_

### How to test changes / Special notes to the reviewer:

### CODE REVIEW

File: renovate.json
Comment (rm3l) [inline renovate.json:5]: Just curious... Will GitHub Action Digests still be pinned via the parent `github>platform-engineering-org/.github`? (Maybe I missed it, but I didn't see this helper being explicitly mentioned in https://github.com/platform-engineering-org/.github/blob/main/default.json)

Comment (rm3l) [reply renovate.json:5]: Ah, I think I found it in https://github.com/renovatebot/renovate/issues/16100. I guess it is part of the `config:best-practices` preset, right?

File: renovate.json
Comment (thepetk) [inline renovate.json:4]: Some thoughts :)

I see inside the `default.json` there is an extension to `schedule:weekly` which is different than our current schedule (we would like to have updates once per sprint). Will this extension affect our current schedule?

Have we tested the new configuration elsewhere to see which will be the behavior of the renovate bot (e.g a forked repo)?

**Review by thepetk — COMMENTED** [review]
I've added some thoughts on the PR

File: renovate.json
Comment (lmilbaum) [reply renovate.json:5]: Correct

File: renovate.json
Comment (lmilbaum) [reply renovate.json:4]: It is used by all the other repos supported by the PE team. For example:
github.com/platform-engineering-org
github.com/arcalot

Comment (lmilbaum) [reply renovate.json:4]: One can override the global settings by adding a setting in the local renovate.json file

**Review by rm3l — APPROVED** [review]

File: renovate.json
Comment (thepetk) [reply renovate.json:4]: I've created this [PR](https://github.com/platform-engineering-org/renovate-runner/pull/95) in order to be able to check how the runner will behave with the global config before merging. @lmilbaum

Comment (thepetk) [reply renovate.json:4]: @lmilbaum thank you for merging the PR I have re-invited the platform-engineering bot in my fork and I'm waiting for the acceptance of the invitation :)

Comment (lmilbaum) [reply renovate.json:4]: @thepetk Done. For some reason GitHub is not sending invite notification emails. Pocking me like you have done is the only way for me to know that such an invite happened. Apologies of the inconvenience.

Comment (thepetk) [reply renovate.json:4]: @lmilbaum I've seen there is a change to the current behavior of the bot.

More detailed due to the packageRule defined [here](https://github.com/platform-engineering-org/.github/blob/main/default.json#L14C4-L20C7) we will now include all devfile image updates inside a single PR (example PR opened here: https://github.com/thepetk/registry/pull/76).

For the registry case, as each stack might be owned by different teams, every devfile image update needs to be inside a different PR. Example PRs:
* https://github.com/devfile/registry/pull/241
* https://github.com/devfile/registry/pull/242

This will allow us to communicate each time with the team responsible for the stack and after their review to merge the PR. If we inherit this package rule we will have to wait for all the teams to review a PR in order to be able to merge it.

I've tried to find a workaround in order to skip this package rule but I wasn't so lucky. Any ideas how we can avoid this rule?

Comment (lmilbaum) [reply renovate.json:4]: @thepetk Apologies for the delay. Too many things on my plate. I think the following should work:
```
  "packageRules": [
    {
      "description": "Disable grouping patch and digest updates",
      "matchUpdateTypes": ["patch", "digest"],
      "groupName": null
    }
]

**Review by thepetk — CHANGES_REQUESTED** [review]
@lmilbaum 

I've checked in my [fork](https://github.com/thepetk/registry/pulls) the updated configuration and it works.

As discussed here https://github.com/devfile/registry/pull/240/files#r1383159668 could you add the following package rule inside the existing list of package rules [here](https://github.com/devfile/registry/blob/33f1ed35a0a803ed111af7ec90e12d4a391c0e5f/renovate.json#L22):

```json
  {
    "description": "Disable grouping patch and digest updates",
    "matchUpdateTypes": ["patch", "digest"],
    "groupName": null
  }
```

File: renovate.json
Comment (thepetk) [reply renovate.json:4]: I'm resolving the issue and proposing the final changes so the PR can be merged!

File: README.md
Comment (thepetk) [inline README.md:50]: @lmilbaum for this section, I like the idea of having a badge for renovate. I'm slightly concerned having a the `badges` section might be misleading for someone as renovate atm only updates the registry stack image versions and not all the dependencies used inside the registry repo.

I would suggest moving the badge:

```
[![Renovate][1]][2]
[1]: https://img.shields.io/badge/renovate-enabled-brightgreen?logo=renovate
[2]: https://renovatebot.com
```

Inside the [regigstry status](https://github.com/devfile/registry/blob/main/README.md#registry-status) section.

**Review by thepetk — CHANGES_REQUESTED** [review]
@lmilbaum as mentioned above the config is fine, I've only added a minor comment regarding the badges section.

File: README.md
Comment (lmilbaum) [reply README.md]: Done

**Review by thepetk — APPROVED** [review]
/lgtm

**Pull request merged**
